tests/issue/109/unit/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("../../../cmake/modules/hunter_boost_component_b2_args.cmake")
+project(TestIssue109Unit)
+
+
+set(component "foo")
+set(cmakeArgs "FOO_OPT=1")
+
+hunter_boost_component_b2_args(
+  "bar"
+  ${cmakeArgs}
+  negativeList
+  )
+
+if(negativeList)
```

Comment:
if value of variable 'negativeList' will be 'NO' we will pass this check. I guess we need `string(COMPARE EQUAL "${negativeList}" "" is_empty)` check